style.css

```diff
@@ -109,8 +109,27 @@ html {
   height: 65px;
 }
 
+/* ABOUTME SECTION */
+.aboutme {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 250px;
+}
+.aboutme__title {
+  font-weight: 400;
+  font-size: 36px;
+}
+.aboutme__video {
+  margin-top: 60px;
+  max-width: 100%;
+  overflow: hidden;
+  cursor: pointer;
+}
```

Comment:
Video looks too small on mobile
![image](https://user-images.githubusercontent.com/68795596/203065214-b14904b0-785b-48d5-a942-2d3b863fbedc.png)

Reply:
Also, just before the tablet breakpoint the video covers the whole screen
![image](https://user-images.githubusercontent.com/68795596/203065861-08ad3ffd-ffc0-460e-8114-cf0035de2d47.png)